apps/framework-cli/src/infrastructure/olap/clickhouse/model.rs

```diff
@@ -109,125 +109,92 @@ pub enum ClickHouseRuntimeEnum {
 }
 
 #[derive(Debug, Clone)]
-pub struct ClickHouseValue {
-    pub value_type: ClickHouseColumnType,
-
-    // This is a string right now because that's the value we send over the wire with the HTTP protocol
-    // if we used the RowBinary // https://clickhouse.yandex/docs/en/query_language/syntax/#syntax-identifiers
-    // or another format, we could optimize
-    pub value: String,
+pub enum ClickHouseValue {
+    String(String),
+    Boolean(String),
+    ClickhouseInt(String),
+    ClickhouseFloat(String),
+    Decimal,
+    DateTime(String),
+    Json,
+    Bytes,
+    Array(Vec<ClickHouseValue>),
+    Enum(String),
+    Nested(Vec<ClickHouseValue>),
+    Null(String),
```

Comment:
Is the enclosed string always `"NULL"`?

Reply:
This is how it would look inside a query 
('T', [('A','B',[('a',[('d','e','f')],'c')],**NULL**)])

Reply:
I have removed the need to pass in a string and instead used the NULL constant